test/hotel_test.rb

```diff
@@ -0,0 +1,282 @@
+
+
+
+require_relative 'test_helper'
+
+
+
+Minitest::Reporters.use! Minitest::Reporters::SpecReporter.new
+
+describe "initialize" do 
+  it "I can access the list of all of the rooms in the hotel" do 
+    hotel = HotelSystem::Hotel.new 
+    hotel.must_be_instance_of HotelSystem::Hotel
```

Comment:
Please use `expect` when writing assertions:

```suggestion
    expect(hotel).must_be_instance_of HotelSystem::Hotel
```

Minitest generates lots of warnings otherwise.  (And in future versions of Minitest this won't work.)